specs/market-api.yaml

```diff
@@ -832,18 +824,39 @@ paths:
 
         **Note**: Mutually exclusive with `rejectAgreement`.
       responses:
-        200:
-          description: Agreement approved.
+        204:
+          description: >
+            Agreement approved.
+
+            Indicates that the approved Agreement has been successfully
+            delivered to the Requestor and acknowledged.
+              - The Requestor side has been notified about the Provider’s commitment.
+              - The Provider is now ready to accept a request to start an Activity.
+              - The Requestor’s corresponding `waitForApproval` call returns `204`
+                (Approved) **after** this endpoint on the Provider side.
         401:
           $ref: 'common.yaml#/responses/Unauthorized'
         404:
           $ref: 'common.yaml#/responses/NotFound'
         408:
           $ref: 'common.yaml#/responses/Timeout'
-        409:
-          description: Agreement not in `Proposal` state.
+          description: Agreement not approved within given timeout. Try again.
```

Comment:
this means that consecutive call to `approveAgreement` is allowed even if previous was successful.